Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby '2.6.3'
```

Comment:
Please check the version. I believe `2.6.5` would make more sense.

Reply:
Sir, I have made the change